pkg/runtime/runtime.go

```diff
@@ -334,6 +332,16 @@ func newDaprRuntime(ctx context.Context,
 		},
 	)
 
+	if runtimeConfig.SchedulerEnabled() {
```

Comment:
Both of these runners are designed to be able to run, regardless of whether there are any addresses/clients. Scheduler is also signaled as enabled if there is a non zero number of addresses. It's not clear to me why we need this change at all.